l1-contracts/deploy-scripts/DeployL1.s.sol

```diff
@@ -217,31 +218,8 @@ contract DeployL1Script is Script, DeployUtils {
         }
     }
 
-    function getRollupL2ValidatorAddress() internal returns (address) {
-        return
-            Utils.getL2AddressViaCreate2Factory(
-                bytes32(0),
-                L2ContractHelper.hashL2Bytecode(L2ContractsBytecodesLib.readRollupL2DAValidatorBytecode()),
-                hex""
-            );
-    }
-
-    function getNoDAValidiumL2ValidatorAddress() internal returns (address) {
-        return
-            Utils.getL2AddressViaCreate2Factory(
-                bytes32(0),
-                L2ContractHelper.hashL2Bytecode(L2ContractsBytecodesLib.readNoDAL2DAValidatorBytecode()),
-                hex""
-            );
-    }
-
-    function getAvailL2ValidatorAddress() internal returns (address) {
-        return
-            Utils.getL2AddressViaCreate2Factory(
-                bytes32(0),
-                L2ContractHelper.hashL2Bytecode(L2ContractsBytecodesLib.readAvailL2DAValidatorBytecode()),
-                hex""
-            );
+    function getRollupL2DACommitmentScheme() internal returns (L2DACommitmentScheme) {
+        return ROLLUP_L2_DA_COMMITMENT_SCHEME;
```

Comment:
does this require a function? its always this constant no? ( or is it overridden somewhere? )

Reply:
I think it was done expecting some polymorphism here. In practice it is excessive. We can remove it if you want

Reply:
not urgent